temp.rs

```diff
@@ -0,0 +1,4 @@
+fn main() {
```

Comment:
This file is useless..

Reply:
woops, im still new at this and i often forget to clean stuff before commiting and pushing